bazel-java-deps.bzl

```diff
@@ -126,7 +126,8 @@ def install_java_deps():
             "org.scalameta:paradise_2.12.6:3.0.0-M11",
             "org.scalameta:scalameta_2.12:1.8.0",
             "org.scala-sbt:sbt:1.1.4",
-            "org.scalatest:scalatest_2.12:3.0.8",
+            "org.scalactic:scalactic_2.12:3.0.5",
+            "org.scalatest:scalatest_2.12:3.0.5",
```

Comment:
These versions now match those introduced within rules_scala internally. Unfortunately, we still have to list these dependencies here even though we're overriding them below. Otherwise the corresponding `@maven//` targets are not generated.